docs/development/language.md

```diff
@@ -0,0 +1,68 @@
+ILIAS Language Handling
+=======================
+ILIAS offers multi-language support for the interface of ILIAS.
+
+# Guidelines
+1.  All language entries are text strings and stored in language files in the 
+    subdirectory `/lang`. Each language entry has the format:
+
+        `language_module_ID#:#variable_ID#:#text_content###comment`
+
+    The elements `language_module_ID`, `variable_ID` and `variable_ID`  are required, 
```

Comment:
Maybe we should use the capitalized keywords `MUST`, `SHOULD`, `REQUIRED` etc., as desribed in [RFC2199](https://www.ietf.org/rfc/rfc2119.txt).

Reply:
Agreed, see comment above.